### PR TITLE
Wire front backend

### PR DESCRIPTION
**Pull request by aymanhab**
This includes the changes to back end side only, specifically:
- add functions to support retrieval of models by id 
- urls to support retrieval of specific models
- models are uploaded kept in folders rather than as individual files to leave place for original moedels/data/meta-data files

>

### CODE REVIEW

**Comment by aymanhab** [comment]
@AlbertoCasasOrtiz for your review

**Review by AlbertoCasasOrtiz — CHANGES_REQUESTED** [review]
LGTM! Only some minor comments. 

Also, It was complaining about axios not found (I think in CI, Jest is complaining about the same). I just had to manually install axios (`npm install axios`) on my computer and it worked.

**Comment by AlbertoCasasOrtiz** [comment]
Hi @aymanhab, I just noticed I didn't told you I reviewed this. The error in jest is because axios is not being automatically installed.

**Comment by AlbertoCasasOrtiz** [comment]
Since it was a while since I reviewed this, I run it locally after manually adding a model to the database and it worked perfectly without having to install anything.

File: src/backend/backend/backend/models.py [hunk 1]
```diff
@@ -10,20 +11,23 @@ class Meta:
     def __str__(self):
         return self.username
 
+def custom_upload_function(instance, filename): 
+    return os.path.join('model_files', str(instance.id), filename)
 
 class Model(models.Model):
```
[the remark this answers is not in the document]
Comment (aymanhab) [reply src/backend/backend/backend/models.py:17]: yes, good idea. Should we hold this PR for it or wait?

**Comment by aymanhab** [comment]
Now that I merged the PR that fixes the backend, not sure if this is ready or not, can you comment @AlbertoCasasOrtiz ? Thank you

**Comment by aymanhab** [comment]
@AlbertoCasasOrtiz can you comment on the status of this PR? Thank you

**Comment by AlbertoCasasOrtiz** [comment]
Hi @aymanhab. There were some conflicts and errors here, but I think I have solved everything. Everything should be working as expected now.

**Comment by aymanhab** [comment]
Perfect, thanks so much @AlbertoCasasOrtiz

File: src/components/pages/OpenSimScene.tsx [hunk 2]
```diff
@@ -34,6 +36,19 @@ const OpenSimScene: React.FC<OpenSimSceneProps> = ({ curentModelPath }) => {
             }),
         [scene]
     )
+    // Lifecycle methods
+    useEffect(() => {
+
+        const getModelGltf = async () => {
+            const result = await axios(
+          'http://localhost:8000/models/viz/default/',
```
[the remark this answers is not in the document]
Comment (AlbertoCasasOrtiz) [reply src/components/pages/OpenSimScene.tsx]: Solved by adding the [] to the useEffect.

**Review by AlbertoCasasOrtiz — APPROVED** [review]

**Comment by AlbertoCasasOrtiz** [comment]
@aymanhab I just noticed I forgot to approve this PR. I just did it.

File: src/components/pages/OpenSimScene.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (aymanhab) [reply src/components/pages/OpenSimScene.tsx]: 👍

**Pull request merged**